package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-autoql",
-  "version": "8.10.16",
+  "version": "8.10.17",
```

Comment:
The next version for his new feature should be v8.11.0 

https://semver.org/#:~:text=MINOR%20version%20when%20you%20add%20functionality%20in%20a%20backward%20compatible%20manner